vertical-pod-autoscaler/recommender/model/aggregate_container_state.go

```diff
@@ -18,45 +18,66 @@ package model
 
 import (
 	"fmt"
+	"math"
 	"time"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	vpa_types "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/apis/poc.autoscaling.k8s.io/v1alpha1"
 	"k8s.io/autoscaler/vertical-pod-autoscaler/recommender/util"
 )
 
+// ContainerNameToAggregateStateMap maps a container name to AggregateContainerState
+// that aggregates state of containers with that name.
+type ContainerNameToAggregateStateMap map[string]*AggregateContainerState
+
 const (
 	// SupportedCheckpointVersion is the tag of the supported version of serialized checkpoints.
+	// Version id should be incremented on every non incompatible change, i.e. if the new
+	// version of the recommender binary can't initialize from the old checkpoint format or the the
+	// previous version of the recommender binary can't initialize from the new checkpoint format.
 	SupportedCheckpointVersion = "v1"
```

Comment:
Shouldn't we bump version with this change?

Reply:
We only checkpoint the VPA state, which should not be affected by this change.